.pre-commit-config.yaml

```diff
@@ -1,48 +1,47 @@
 repos:
```

Comment:
Nit: For future reference probably would have been good to do the isort update in a separate PR to avoid confusing this one.

Reply:
Sorry I was mostly just trying to get this to stop screaming and didn't really think about that. Want me to open a separate PR for this so it can go into main while this is still a WIP?

Reply:
No, no. It's fine, just an FYI for future tickets. Better to have lots of small self contained PRs than one bigger one